internal/configs/testdata/invalid-modules/provider-meta-invalid-name/main.tf

```diff
@@ -0,0 +1,5 @@
+terraform {
```

Comment:
I see this new file has been added but is a test already referencing it? I couldn't find one but maybe im blind.

Reply:
Took me a while as well. There's a magical test that picks up all the directories in the `invalid-modules` directory and runs them as individual test cases.

Reply:
Oh awesome :D